wikidata_game/composer.json

```diff
@@ -10,5 +10,8 @@
     "license": "GPLv3",
     "require": {
       "ext-pdo": "*"
+    },
+    "autoload": {
+        "psr-4": {"ReferenceIsland\\": "./"}
```

Comment:
I think if we're using PSR 4 then the directory structure needs to match the namespace paths, right?

Reply:
I *think* that the "base-directory" is the only one that doesn't need to match the namespace paths. Any subnamespaces (and the file name for each class) however should match.

We could also consider sticking everything in an `src` if you'd prefer

https://www.php-fig.org/psr/psr-4/ is where a found a semi-human readable explaination